src/Platform/Hosted/extractFeatures.ts

```diff
@@ -55,6 +56,8 @@ export function extractFeatures(given = new URLSearchParams(window.location.sear
     enableSpark: "true" === get("enablespark"),
     enableTtl: "true" === get("enablettl"),
     executeSproc: "true" === get("dataexplorerexecutesproc"),
+    fetchesAADToken: true,
+    // fetchesAADToken: "true" === get("fetchesaadtoken"),
```

Comment:
for testing

Reply:
Uncomment this. We can't deploy this to MPAC